gaia-list.js

```diff
@@ -256,15 +256,6 @@ module.exports = component.register('gaia-list', {
       width: auto;
     }
 
-    /**
-     * Reverse the icons when the document is RTL mode
-     */
-
-    :host-context([dir=rtl]) ::content i:before {
-      transform: scale(-1, 1);
-      text-align: end;
-    }
-
```

Comment:
I don't like making the assumption that all arrows should be flipped in RTL mode. This isn't obvious as a user of gaia-icons and can introduce bugs. I'd much rather we do it on a case-by-case basis :) So let's put the rules in gaia-list.